docs/website/versioned_docs/version-3.0.0/user-guides/quickstart.md

```diff
@@ -5,7 +5,13 @@ sidebar_position: 5
 
 # Quickstart Guide
 
-In this guide, we will be using odo to create a "Hello World" application with Node.js
+In this guide, we will be using odo to create a "Hello World" application.
+
+You have the option of choosing from the following frameworks for the quickstart guide:
+* Node.js
+* .NET
+
+A full list of example applications can be viewed with the `odo registry` command.
```

Comment:
Think about this as a user. If I want to see example applications, how does it make sense to run a `registry` command? IMO, we are having a terrible UX where commands don't map to the actual thing. This is not feedback for this PR, but something that we should really think about when revamping odo to v3. @kadel @deboer-tim @serenamarie125